lib/mjsonwp/routes.js

```diff
@@ -9,7 +9,9 @@ const METHOD_MAP = {
     GET: {command: 'getStatus'}
   },
   '/wd/hub/session': {
-    POST: {command: 'createSession', payloadParams: {required: ['desiredCapabilities'], optional: ['requiredCapabilities', 'capabilities']}}
+    POST: {command: 'createSession',
+           payloadParams: {required: ['desiredCapabilities'],
+                           optional: ['requiredCapabilities', 'capabilities', 'alwaysMatch' , 'firstMatch']}}
```

Comment:
pretty sure that a new client will send in `capabilities` _instead of_ `desiredCapabilities`, and further that `alwaysMatch` and `firstMatch` will be properties of the `capabilities` object, not top-level properties.

not sure if we have a way to specify in base driver that as a requirement we want simply one of a set, rather than all of a set...